addon/-cancelable-promise-helpers.js

```diff
@@ -116,10 +116,12 @@ function taskAwareVariantOf(obj, method, getItems) {
       if (hasCancelled) { return; }
       hasCancelled = true;
       items.forEach(it => {
-        if (it instanceof TaskInstance) {
-          it.cancel();
-        } else if (typeof it.__ec_cancel__ === 'function') {
-          it.__ec_cancel__();
+        if (it !== null && it !== undefined) {
```

Comment:
I decided to check this way rather than just a truthy check for a pedantic and maybe-silly reason. Theoretically somebody could provide a non-`null` non-`undefined` falsey value with an `__ec_cancel__` property on it. I can't imagine why somebody would want to, but I also can't see a reason why we shouldn't call such a method, so I made the check stricter.